packages/office-ui-fabric-react/src/components/Rating/Rating.Props.ts

```diff
@@ -53,6 +54,13 @@ export interface IRatingProps extends React.HTMLProps<HTMLElement> {
    * Optional id of label describing this instance of Rating
    */
   ariaLabelId?: string;
+
+  /**
+   * Optional setting to represent rating control as aggregate rating control.
+   * If set, rating control will be readonly rating control with partial star support
+   * Also if set, this overrides the disabled prop
+   */
+  aggregate?: boolean;
```

Comment:
This comment is quite confusing for documentation (this will go into the official documentation.)

What is an aggregate rating control?

If you have a read-only mode, why not make that an explicit prop: `readOnly?: boolean`

Why would it override the disabled prop? That also seems overloaded.